### PR TITLE
setup structure

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
## What does this PR do
In this PR me and my coding partner check the error about the feature and make the changes.
## Description of the task to be completed
- [x] Setup project structure
- [x] setup webpack 

 
## How should it be tested
Clone the repository
```
https://github.com/AhmedMohsen600/to-do-list.git
```
Navigate to the folder
```
cd to-do-list
```
git checkout feature
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import './style.css';
 import '@fortawesome/free-solid-svg-icons';
+
 const todo = ['go to gym', 'Study', 'work'];
```
Comment (levy002) [inline src/index.js]: - [x] Kindly make sure your task is an object containing three keys - description \[`string`\], completed \[`boolean`\], index: \[`number`\].
 
  _**Example:**_

            {
                description: 'An example described in the string',
                completed: false,
                index: 10,
            }

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @AhmedMohsen600   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (AhmedMohsen600) [reply src/index.js]: hello @levy002  thank you for your comment should I name it index? or can I name it Id cuz id makes more sense to me

**Review by Ismailco — APPROVED** [review]
### Hi @AhmedMohsen600, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**